src/Admin/AdminInterface.php

```diff
@@ -135,12 +135,7 @@ public function getSecurityInformation(): array;
 
     public function setParentFieldDescription(FieldDescriptionInterface $parentFieldDescription): void;
 
-    /**
-     * Get parent field description.
-     *
-     * @return FieldDescriptionInterface The parent field description
-     */
-    public function getParentFieldDescription(): ?FieldDescriptionInterface;
+    public function getParentFieldDescription(): FieldDescriptionInterface;
```

Comment:
This change is consistent with the implementation and avoid an error on max level of phpstan

Reply:
Does the implementation throw a exception if there is no value ? We could add the phpdoc.

Reply:
It does, will add it

Reply:
Done